CRM/Contact/BAO/Relationship.php

```diff
@@ -271,7 +271,7 @@ public static function addRecent($params, $relationship) {
     if (($session->get('userID') == $relationship->contact_id_a) ||
       CRM_Contact_BAO_Contact_Permission::allow($relationship->contact_id_a, CRM_Core_Permission::EDIT)
     ) {
-      $rType = substr(CRM_Utils_Array::value('relationship_type_id', $params), -3);
+      $rType = substr($params['relationship_type_id'], -3);
```

Comment:
Would this create a notice? or is it def set?

Reply:
If it wasn't set, it would be a bug, and `CRM_Utils_Array::value` would have masked the error.